### ISSUE TITLE
Add a pager status message

### ISSUE THREAD

**Issue by aleris** [Tradeshift/tradeshift-ui#853]
**Is your feature request related to a problem? Please describe.**
For Document Manager redesign we need to show the current documents and the total number of documents like below:
![image](https://user-images.githubusercontent.com/1011245/63263784-31eadc80-c292-11e9-981d-77ec1dc8498b.png)

**Describe the solution you'd like**
Send the information to be shown on Pager model, something like:

```
.pager({
    pages: 5,
    page: 0,
    status: '1-5 of 25'
})
```

**Describe alternatives you've considered**
Show the information on normal status bar, however this has the disadvantage of taking vertical space.

**Additional context**
See:
https://tradeshift.invisionapp.com/share/CUSV8MA9EVA#/screens/372665844
PAYDOCS-19

**Comment by esmailpour-hosein** [comment]
I think it should be a boolean(show status), to either show the status or not to keep it consistent over the platform! Maybe only the type of the items?  Do you see any other case that the status message needs customization? 
@mihneaz do you think it's okay if we replace the items type (in this case `documents`) with something more generic like `results` or `items`? In this way it's always looks the same and we are responsible for the translation and knowing the range of characters that we should expect for that part for the matter of responsiveness.

**Comment by esmailpour-hosein** [comment]
#846

**Comment by mihneaz** [comment]
@esmailpour-hosein Yes, let's keep it generic (`items`). `Results` would also work but it implies an active query (search or set of filters applied).

**Comment by zdlm** [comment]
Implemented in V 12.3.1
https://ui.tradeshift.com/v12/#components/table/paging.html

**Comment by aleris** [comment]
Thanks for that. Integrated and works ok.

One small problem is that the `number` is not limited to total, for example if we have 20 per page and only 5 items in total, the pager will display 1-20 (5). However is pretty straightforward as a workaround for this small issue to calculate `number` and limit it to the actual total items before send it to pager, eg:

`const numberUpToTotal = Math.min(total, number) || 0;`

and then 
```
.pager({
  pages: pages,
  page: page,
  number: numberUpToTotal,
  total: total...
```

**Comment by zdlm** [comment]
Nice catch. 
It is a bug. I will fix it later.
Thank.

**Comment by aleris** [comment]
If we have just one item in the second page should it display '21 - 21 (21)'?

![image](https://user-images.githubusercontent.com/1011245/65759710-32de1c00-e124-11e9-8ebe-abf4d3af15b9.png)

**Comment by esmailpour-hosein** [comment]
> If we have just one item in the second page should it display '21 - 21 (21)'?

Yes

**Comment by aleris** [comment]
Sorry for not being very explicit. I displays 21-40 with 21 total items (see screenshot) so the upper page limit should also be limited to the total.

**Comment by esmailpour-hosein** [comment]
NP, I think it will be after the fix PR! I'll make sure that it works in this scenario. Thanks 👍
